easybatch-jdbc/src/main/java/org/easybatch/jdbc/JdbcRecordWriter.java

```diff
@@ -68,6 +68,7 @@ public void writePayload(final Object record) throws RecordWritingException {
             PreparedStatement preparedStatement = connection.prepareStatement(query);
             preparedStatementProvider.prepareStatement(preparedStatement, record);
             preparedStatement.executeUpdate();
+            preparedStatement.close();
```

Comment:
It would be all right. I have to tried by overreding it because I was unable to patch it directly withou rebuilding everything.